Description: The list value should have a semicolon (';') as trailing character, and fixing lintian tags.
Origin: upstream, https://github.com/fourdollars/betaradio/commit/d39541d0f13703e6103be30b3e35ded4c671bcdf
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: betaradio-1.6/data/betaradio.desktop.in
===================================================================
--- betaradio-1.6.orig/data/betaradio.desktop.in	2014-08-18 11:46:32.000000000 +0800
+++ betaradio-1.6/data/betaradio.desktop.in	2014-08-18 15:35:50.498429844 +0800
@@ -1,5 +1,6 @@
 [Desktop Entry]
-Categories=GTK;AudioVideo;Audio;Tuner;Music;TrayIcon
+Categories=GTK;AudioVideo;Audio;Tuner;Music;TrayIcon;
+OnlyShowIn=GNOME;
 Exec=betaradio
 Icon=betaradio
 Type=Application
@@ -8,3 +9,5 @@
 Name[zh_TW]=BetaRadio 點播器
 Comment=An easy way to listen to internet radio of Taiwan.
 Comment[zh_TW]=用簡單的方式收聽台灣網路電台
+Keywords=radio;taiwan;
+Keywords[zh_TW]=廣播;台灣;
